manifests/agent.pp

```diff
@@ -310,15 +300,35 @@
     }
   }
   ini_setting {'puppetagentconfigtimeout':
-    ensure  => present,
     setting => 'configtimeout',
     value   => $configtimeout,
   }
   if $stringify_facts != undef {
     ini_setting {'puppetagentstringifyfacts':
-      ensure  => present,
       setting => 'stringify_facts',
       value   => $stringify_facts,
     }
   }
+  if $serialization_format != undef {
```

Comment:
Are these already packaged for use

libmsgpack-ruby1.8 - binary-based efficient object serialization library for ruby1.8
libmsgpack-ruby1.9.1 - binary-based efficient object serialization library for ruby1.9.1